### PR TITLE
docs: remove await outside of an async function in example

### PR DESCRIPTION
**Pull request by RichardLitt**
See #549.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ const OrbitDB = require('orbit-db')
 
 const ipfs = IpfsApi('localhost', '5001')
 const orbitdb = new OrbitDB(ipfs)
-const db = await orbitdb.log('hello')
+const db = orbitdb.log('hello')
```
Comment (shamb0t) [inline README.md]: `orbitdb.log` is an async function so it will return a promise here. Maybe `orbitdb.log('hello').then(db => // do stuff with db )` ?

**Comment by shamb0t** [comment]
Thanks @RichardLitt! You're right the issue is the three lines in the example should be called within an async function. But `.log` is async and will return a promise

**Comment by RichardLitt** [comment]
Thanks, Shams. How does this look?

File: README.md [hunk 2]
```diff
@@ -139,8 +139,9 @@ const OrbitDB = require('orbit-db')
 
 const ipfs = IpfsApi('localhost', '5001')
 const orbitdb = new OrbitDB(ipfs)
-const db = await orbitdb.log('hello')
-...
+const db = orbitdb.log('hello').then(db => {
```
Comment (shamb0t) [inline README.md]: The call will still return a promise so I think we should get rid of the `const db = `

**Comment by shamb0t** [comment]
@RichardLitt one last comment! :D

File: README.md (hunk 2 above)
Comment (RichardLitt) [reply README.md]: Removed!

**Comment by shamb0t** [comment]
Thank you @RichardLitt ! :+1:

**Pull request merged**